lib/testRun.js

```diff
@@ -63,6 +63,11 @@ module.exports = function() {
       if (text.match(/^[a-z0-9]+$/)) {
         annotation.key = text;
       } else {
+        var active = parseAnnotationValue(text.toLowerCase(), /active\=[\"\']?([1|0|true|false|yes|no])[\"\']?/);
+        if (active) {
+          annotation.active = active === 'true' || active === '1' || active === 'yes';
```

Comment:
If you make the changes described above, I would replace this with:

`annotation.active = active.match(/^(1|y|yes|t|true)$/i);`